pkg/client/vcwallet/client_test.go

```diff
@@ -372,10 +414,20 @@ func TestClient_Get(t *testing.T) {
 	require.NotEmpty(t, vcWalletClient)
 	require.NoError(t, err)
 
-	result, err := vcWalletClient.Get("")
-	require.Empty(t, result)
+	err = vcWalletClient.Add(Metadata, []byte(sampleContentValid))
+	require.NoError(t, err)
+
+	content, err := vcWalletClient.Get(Metadata, "did:example:123456789abcdefghi")
+	require.NoError(t, err)
+	require.NotEmpty(t, content)
+
+	err = vcWalletClient.Remove(Metadata, "did:example:123456789abcdefghi")
+	require.NoError(t, err)
+
+	content, err = vcWalletClient.Get(Metadata, "did:example:123456789abcdefghi")
+	require.Empty(t, content)
 	require.Error(t, err)
-	require.EqualError(t, err, toBeImplementedErr)
+	require.Contains(t, err.Error(), "data not found")
```

Comment:
Consider using `errors.Is(spi.ErrDataNotFound)` (if possible)

Reply:
this is unit test and this is "contains" check

Reply:
You can also check for it by using `errors.Is(spi.ErrDataNotFound)`, which is an even better check since it will avoid false positives and will ensure that even if the error message changes, that the test will still pass

Reply:
You don't need to check the whole error this way either. You don't want some other "data not found" string to accidentally pass it.